datasets/lt/illegal_gambling/crawler.py

```diff
@@ -0,0 +1,35 @@
+import re 
+
+from zavod import Context
+
+def crawl_item(item, context: Context):
+
+    name = item.findtext(".//td[2]")
+    domain = item.findtext(".//td[3]")
+    contacts = item.findtext(".//td[4]")
+
+    if not domain:
+        return
+
+    entity = context.make("Organization")
+    entity.id = context.make_slug(domain, prefix="lt-illegal-gambling")
```

Comment:
dataset prefix is used automatically, isn't it?
```suggestion
    entity.id = context.make_slug(domain)
```